http/src/main/resources/reference.conf

```diff
@@ -251,6 +251,11 @@ gke {
     postgresDiskSizeGB = 10
     postgresDiskBlockSize = 4096
   }
+  customApp {
```

Comment:
you can also make these into kebab case and use pureconfig instead...and then just need to add it [here](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/ConfigReader.scala#L18)